package.json

```diff
@@ -72,7 +72,7 @@
   "peerDependencies": {
     "react": "*",
     "react-native": "*",
-    "react-native-vision-camera": ">=2.16.4"
+    "react-native-vision-camera": ">=3.6.17"
```

Comment:
What this intended? Now we are tracking v3 for the peer. I believe this package is v2 only.

Reply:
oh yeah must be remnants from when I tried v3 last time